src/Spec2-Core/SpToolbarDisplayModeIconAndLabel.class.st

```diff
@@ -17,7 +17,7 @@ SpToolbarDisplayModeIconAndLabel >> configureButton: aButton item: aToolbarItem
 
 { #category : #accessing }
 SpToolbarDisplayModeIconAndLabel >> extent [
-	^ 45@40
+	^ 45@45
```

Comment:
I do not like this one. 40 is already very big. Why is this needed?

Reply:
To squared. if not we can use 40@40

Reply:
mmm... ok ?
let´s test it.